examples/5d8nc8tp93_glassplate/iiif-frame-glassplate-manifest.json

```diff
@@ -0,0 +1,150 @@
+{
+  "@type": "Manifest",
+  "@embed": "@always",
+  "@context": {
```

Comment:
Is het geen probleem dat die context wordt over gekopieerd? Mogelijk gaat het ontbreken van de http://iiif.io/api/presentation/3/context.json url voor problemen zorgen, zie https://iiif.io/api/presentation/3.0/#46-linked-data-context-and-extensions.

Ik zou ook een ticket aanmaken voor triply op support@triply.cc over die `Invalid JSON-LD syntax; invalid scoped context.` bij het ophalen van externe contexts in JSON-LD frames.